app/src/test/scala/org/alephium/explorer/Generators.scala

```diff
@@ -132,10 +133,34 @@ object Generators {
     key  <- hashGen
   } yield protocolApi.OutputRef(hint, key)
 
+  lazy val keypairGen: Gen[(PrivateKey, PublicKey)] =
+    Gen.const(()).map(_ => SignatureSchema.secureGeneratePriPub())
```

Comment:
Just checking the use-case for a `lazy val` here. A normal `val` should be good here right?

Reply:
totally.....I copy pasted this from `alephium` and didn't check further :see_no_evil: